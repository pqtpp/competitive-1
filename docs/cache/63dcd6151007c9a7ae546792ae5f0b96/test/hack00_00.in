2
4033
4681
