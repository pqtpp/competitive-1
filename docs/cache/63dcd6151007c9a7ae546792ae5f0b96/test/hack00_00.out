2 37 109
2 31 151
